src/main/java/it/pagopa/interop/probing/eservice/registry/updater/dto/EserviceDTO.java

```diff
@@ -0,0 +1,105 @@
+/**************************************************************************
+*
+* Copyright 2023 (C) DXC
+*
+* Created on  : 7 mar 2023
+* Author      : dxc technology
+* Project Name: interop-be-probing-eservice-registry-updater 
+* Package     : it.pagopa.interop.probing.eservice.registry.updater.dto
+* File Name   : EserviceDTO.java
+*
+*-----------------------------------------------------------------------------
+* Revision History (Release )
+*-----------------------------------------------------------------------------
+* VERSION     DESCRIPTION OF CHANGE
+*-----------------------------------------------------------------------------
+** --/1.0  |  Initial Create.
+**---------|------------------------------------------------------------------
+***************************************************************************/
+package it.pagopa.interop.probing.eservice.registry.updater.dto;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import it.pagopa.interop.probing.eservice.registry.updater.annotations.ValidateEnum;
+import it.pagopa.interop.probing.eservice.registry.updater.annotations.ValidateStringArraySize;
+import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceState;
+import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceTechnology;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * Gets the producer name.
```

Comment:
I guess that the comment is wrong (same for the setter)

Reply:
I deleted the comments, then we will try to fix or delete the jautodoc tool

Reply:
still there